daffodil-runtime1/src/main/scala/org/apache/daffodil/runtime1/debugger/InteractiveDebugger.scala

```diff
@@ -1663,9 +1663,9 @@ class InteractiveDebugger(
         val desc = "display differences since the previous pause in the debugger"
         val longDesc = desc
 
-        lazy val infoDiffables = Info.subcommands.collect { case diffable: InfoDiffable =>
-          diffable
-        }
+        lazy val infoDiffables = Info.subcommands
+          .filter(_.isInstanceOf[InfoDiffable])
+          .map(_.asInstanceOf[InfoDiffable])
```

Comment:
This seems like worse syntax, does scala not support this use of `collect` anymore?

Reply:
So you can use collect, but because InfoDiffable is an inner trait, it doesn't know the type and causes this error:

The outer reference in this type test cannot be checked at run time.
[error]         lazy val infoDiffables = Info.subcommands.collect { case diffable: InfoDiffable =>